src/kbc_modules/offline_fs_kbc/common.rs

```diff
@@ -82,6 +102,31 @@ pub mod tests {
         keyfile_path
     }
 
+    #[allow(dead_code)]
```

Comment:
I think you can avoid this dead code warning if you move this function out of the test module and then import it.

Reply:
this function is for test only, so maybe let's keep it in test module?